internal/pkg/externalplugins/labelblocker/labelblocker.go

```diff
@@ -0,0 +1,186 @@
+package labelblocker
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/ti-community-infra/tichi/internal/pkg/externalplugins"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/test-infra/prow/config"
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/pluginhelp"
+)
+
+const PluginName = "ti-community-label-blocker"
```

Comment:
Myabe ti-comunity-labelblocker is better?

Reply:
But they're not a combination, IDE will prompt typo.

Reply:
Got it, then do not change it.